setup.py

```diff
@@ -24,13 +33,14 @@
     ],
     extras_require={
         'dev': [
-            "coverage==4.0a1",
+            "coverage==4.0b3",
```

Comment:
Thanks. I added this to #250 and cherry-picked it here. If you're happy with that branch can you give it a :+1: ?